selenium/che-selenium-core/bin/webdriver.sh

```diff
@@ -142,6 +143,7 @@ checkParameters() {
         elif [[ "$var" =~ -P.* ]]; then :
         elif [[ "$var" == "--help" ]]; then :
         elif [[ "$var" == "--compare-with-ci" ]]; then :
+        elif [[ "$var" =~ --workspace-pool-size=auto|[0-9]+$ ]]; then :
```

Comment:
Have you tried both following variants?
`--workspace-pool-size=auto`
`--workspace-pool-size=2`
I'm not sure that regexp is correct. Can you check whether it should be something like:
`^--workspace-pool-size=((auto)|([0-9]+))$`

Reply:
I have tried :-)

Reply:
The reason is that `--workspace-pool-size=aut1` also matches this regexp.

Reply:
Do you think it is worth to fix?

Reply:
:-) good catch